docs/UserGuide.md

```diff
@@ -16,6 +16,7 @@
     * [`author-config.csv`](#author-configcsv)
 * [Analyzing Multiple Repos](#analyzing-multiple-repos)
 * [Using Travis-CI to automate publishing of the report to GitHub Pages](#using-travis-ci-to-automate-publishing-of-the-report-to-github-pages)
+* [Note on Binary Files](#note-on-binary-files)
```

Comment:
Seems weirder if the part is here. Let the seniors decide where to place them.

Reply:
I think can remove the entire topic on `Note on Binary Files` for now and just leave it in FAQ